extensions-core/multi-stage-query/src/test/java/org/apache/druid/msq/exec/MSQSelectTest.java

```diff
@@ -1641,8 +1622,13 @@ public void testTimeColumnAggregationFromExtern() throws IOException
                 + "FROM kttm_data "
                 + "GROUP BY 1")
         .setExpectedValidationErrorMatcher(
-            ThrowableMessageMatcher.hasMessage(CoreMatchers.containsString(
-                "LATEST() aggregator depends on __time column"))
+            new DruidExceptionMatcher(DruidException.Persona.ADMIN, DruidException.Category.INVALID_INPUT, "adhoc")
+                .expectMessageIs(
+                    "Query planning failed for unknown reason, our best guess is this "
```

Comment:
I'm anti that messaging because it pretends like the following error message has a high likelihood of being the cause.  My experience has told me that this is rarely the case and most likely this error message means "something happened, I'm gonna tell you something that is just as likely to be wrong as it is right, good luck".  So I don't want to pretend like it's giving people a good answer when it generally is not.